swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -5298,6 +5339,67 @@ components:
       example: ["Ref Number Merchant", "Some Other Text"]
 
 
+    EntryDetailsElement:
```

Comment:
Relevant Change: EntryDetailsElement is new, we have to check if the data types match what we need.
Adoption approach: Follow the change

Reply:
From my point of view, the element `EntryDetailsElement` is not new. In the [previous version](https://github.com/openbankingproject-ch/obp-apis/blob/master/swiss-ng-api/berlin-group-documentation/psd2-api%201.3.8%202020-12-14v2.yaml) I could find it on line 5335. 

The impact would probably be rather small anyway, since it is optional and currently most likely not used.

Reply:
already in there